lua/telescope/pickers.lua

```diff
@@ -756,6 +760,14 @@ function Picker:reset_selection()
   self.multi_select = {}
 end
 
+function Picker:refresh(finder)
+  local pbnr = self.prompt_bufnr
+  vim.api.nvim_buf_set_lines(pbnr, 0, 1, false, {})
+  vim.api.nvim_buf_add_highlight(pbnr, ns_telescope_prompt_prefix, 'TelescopePromptPrefix', 0, 0, #self.prompt_prefix)
```

Comment:
Doe we already have some code that does this? We should call the same function if so.

Reply:
It lays somewhere above 😆 we could do a local function for it